articles/multifactor-authentication/overview.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Multi-factor Authentication in Auth0
+description: The basics of multi-factor authentication and the different methods of implementing it with Auth0.
+url: /multifactor-authentication
+toc: true
+topics:
+    - mfa
+contentType:
+  - index
+useCase:
+  - customize-mfa
+---
+# Multi-factor Authentication
+
+## What is multi-factor authentication?
+
+Multi-factor Authentication (MFA) is a method of verifying a user's identity by requiring them to present more than one piece of identifying information. This method provides an additional layer of security, decreasing the likelihood of unauthorized access. The type of information required from the user is typically two or more of the following:
+
+* **Knowledge**: Something the user **knows** (such as a password)
+* **Possession**: Something the user **has** (such as a mobile device)
+* **Inheritance**: Something the user **is** (such as a fingerprint or retina scan)
+
+## Implement MFA with Auth0
+
+Enabling MFA for your tenant is a fairly straightforward process. First, you toggle on the factors you choose to enable on your tenant, such as push notifications or SMS. Next, you perform any further setup required to configure that factor, and last, you choose whether you wish to force MFA for all users or not. See the instructions below for details.
+
+You can also [customize your MFA flow](/multifactor-authentication/custom) with [Auth0 Rules](/rules), to allow MFA to only be required in specific circumstances or force a particular factor to be used.
+
+### 1. Toggle on the factors you require in the Dashboard
+
+In the [Dashboard > Multifactor Auth](${manage_url}/#/mfa), head to the Multifactor Auth section. Here you will find a series of toggles for the MFA factors supported by Auth0. 
+
+**TODO: STANDARDIZE DASHBOARD IMAGE**
+
+![MFA Dashboard Page](/media/articles/multifactor-authentication/mfa-dashboard.png)
+
+Any or all of these factors can be enabled simultaneously. When logging in the first time, the user will be shown a default and but will be allowed to choose another factor to use, if you have more than one factor enabled in the Dashboard. 
+
+**TODO: IMAGE OF CURRENT LOGIN PROMPT WITH MFA CHOICES**
+
+::: note
+When you enable the Duo factor, you will have to fill in a few further settings related specifically to Duo before continuing.
+**TODO: SPECIFY SETTINGS**
+:::
+
+Additionally, there is the **Always require Multi-factor Authentication** toggle, which allows authentication against this app only when one or more of the factors is enabled in the Dashboard, and completed by users while logging in.
```

Comment:
I think we might need to rephrase this one, 'this app' is not clear to me at least. 

Maybe the title could be something like 'Toggle on the factors you want to support' and then 'Require MFA for your applications' with an explanation of what the `Always require Multi-factor Authentication` toggle does?